source/includes/swift/_objects.md

```diff
@@ -0,0 +1,24 @@
+### Objects
+
+Create and manage your objects.
+
+<!-------------------- PREVIEW OBJECTS -------------------->
+
+#### Preview objects
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/swift/my-env/objects/my-private-bucket/my-object.jpg/redirect?operation=preview"
```

Comment:
why `/redirect`?

Reply:
As per the [ServiceEntityResource](https://github.com/cloudops/cloudmc-core/blob/devel/src/main/java/com/cloudops/mc/rest/resource/services/ServiceEntityResource.java#L546) in core, looks like you need to make a GET to an endpoint that ends in `/redirect` to perform a redirect operation

Cloustack's "open console" is the only other operation that uses redirects as far as I know but there are no docs on this 👀